.luacheckrc

```diff
@@ -0,0 +1,13 @@
+unused_args = false
+allow_defined_top = true
+
+read_globals = {
+	"DIR_DELIM",
+	"vector", "nodeupdate",
+	"VoxelManip", "VoxelArea",
+	"PseudoRandom", "ItemStack",
+}
+globals = { "minetest", "core" }
```

Comment:
Not sure, why do you think it should be discouraged?
It's a reference to the same table and not going anywhere.